api/src/main/java/org/openmrs/api/cache/CacheConfig.java

```diff
@@ -16,7 +16,9 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.core.io.ClassPathResource;
 
-
+/**
+ *@see <a href="https://wiki.openmrs.org/display/docs/For+Module+Developers?">https://wiki.openmrs.org/display/docs/For+Module+Developers?</a>
```

Comment:
The page to link is: https://wiki.openmrs.org/display/docs/Caching+service+methods, but use tiny a link instead https://wiki.openmrs.org/x/IYaEBg which can be generated from wiki by clicking at three dots in the top right corner and selecting link to this page.

The doc should be more helpful... something like:

"CacheConfig provides a cache manager for the @Cacheable annotation and uses ehCache under the hood. The config of ehCache is loaded from ehache-api.xml and can be extended by modules through apiCacheConfig.properties. For more details see the wiki page at https://wiki.openmrs.org/x/IYaEBg"